test/fixtures/netweaver_simple_example/files/templates/post_deployment_script.tpl

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
+sudo su -
```

Comment:
I don't think this script will work. `sudo su -` will start a new login shell and wait for commands; the additional commands in this script will not be executed until the login shell exits.

Reply:
removed the gap between #!  /bin/bash . removed the sudo su - command too .